02_Development/Display/moving_average_filter.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+from numpy_ringbuffer import Ringbuffer
+
+
+class MovingAverageFilter(Ringbuffer):
+
+
+	def __init__(self, size):
+
+		super().__init__(size)
+
+	def get_datum(self):
+
+		return np.mean(self)
```

Comment:
this class seems... incomplete.  How big is the buffer?

Reply:
This was in fact incomplete and only got added on accident.  But it's a simple class anyway so I've added more detail and tests, so you'll see that in the next commit.